utils/s2n_safety.h

```diff
@@ -91,6 +91,7 @@ static inline void* trace_memcpy_check(void *restrict to, const void *restrict f
 #define GUARD( x )              if ( (x) < 0 ) return -1
 #define GUARD_GOTO( x , label ) if ( (x) < 0 ) goto label
 #define GUARD_PTR( x )          if ( (x) < 0 ) return NULL
+#define S2N_IN_UNIT_TEST ( getenv("S2N_UNIT_TEST") != NULL )
```

Comment:
Does getenv need an include for stdlib?

Reply:
I added one for clarity/safety, something else must have already included it so it was just working.